modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -980,11 +983,11 @@ private void assertTablePresent(TableManager tableManager, String tableName) {
     public void testOneNodeRestartWithGap() throws InterruptedException {
         IgniteImpl ignite = startNode(0);
 
-        List<IgniteComponent> components = startPartialNode(1, null);
+        startNode(1);
```

Comment:
Why it's full node instead of partial?

Reply:
Please address given question. You may create separate ticket if needed.

Reply:
Added TODO https://issues.apache.org/jira/browse/IGNITE-19408